include/btllib/aahash_consts.hpp

```diff
@@ -0,0 +1,2582 @@
+#ifndef BTLLIB_AAHASH_CONSTS_HPP
+#define BTLLIB_AAHASH_CONSTS_HPP
+
+#include <cmath>
+#include <cstdint>
+#include <vector>
+
+#include "btllib/nthash_consts.hpp"
+// clang-format off
```

Comment:
I think the only complaints we got from clang-format was the double underscore issue and using `static` in a namespace. We can remove the `static`s and it won't create any issues. If that's it then let's not turn clang-format off for this file.

Reply:
Using Vlad's suggestion, I localized clang-format off/on to lines `529-544`.